src/Models/FileLink.php

```diff
@@ -18,8 +19,14 @@ class FileLink extends Link
         'File' => File::class,
     ];
 
+    /**
+     * @deprecated 3.0.0 Will be removed without equivalent functionality to replace it
+     */
     public function generateLinkDescription(array $data): string
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('3.0.0', 'Will be removed without equivalent functionality to replace it.');
```

Comment:
```suggestion
     * @deprecated 3.0.0 Use getDescription() instead.
     */
    public function generateLinkDescription(array $data): string
    {
        Deprecation::withNoReplacement(function () {
            Deprecation::notice('3.0.0', 'Use getDescription() instead.');
```
This serves the same purpose as `getDescription()`.
That method doesn't exist but this is the language we used for the 4->5 deprecations even when that method didn't exist in 4..... probably something like "Will be replaced with getDescription()" would be better though if you'd prefer that.

Reply:
Updated